sizes/output.go

```diff
@@ -279,10 +279,10 @@ func (t *Threshold) Type() string {
 // A `pflag.Value` that can be used as a boolean option that sets a
 // `Threshold` variable to a fixed value. For example,
 //
-//		pflag.Var(
-//			sizes.NewThresholdFlagValue(&threshold, 30),
-//			"critical", "only report critical statistics",
-//		)
+//	pflag.Var(
+//		sizes.NewThresholdFlagValue(&threshold, 30),
+//		"critical", "only report critical statistics",
+//	)
```

Comment:
Is this change fixing something?  If not, can we revert it?

Reply:
This is just the result of vscode-go's automatic formatting

Reply:
Okay, then let's revert this.  Again, it's better not to make extraneous changes in one commit, since that helps us determine what changes are logically necessary.

If you could, please squash the reverts down into the original commit and force-push over the branch.